src/github/components/GithubObject.js

```diff
@@ -205,42 +205,41 @@ class GithubObject {
       json: true,
     };
 
-    return req(config).then((response) => {
-      if (response.body instanceof Array) {
-        // Default GET Github API requests.
-        results.push(...response.body);
-      }
-      else if (response.body instanceof Object) {
-        // Default single GET Github API requests.
-        results.push(response.body);
-      }
-      else if (response.body.items instanceof Array) {
-        // Support SEARCH Github API requests.
-        // @link https://developer.github.com/v3/search/
-        results.push(...response.body.items);
-      }
-      else {
-        log('reject');
-      }
+    return axios(config)
+      .then((response) => {
+        if (response.body instanceof Array) {
```

Comment:
Did you test this? I thought a successful Axios request returns a thing with a `data` property, not `body` (ref https://github.com/axios/axios/blob/f3ca6371caa738ba5308d413433d9f676f2e0138/index.d.ts#L84)

Reply:
Yes I agree there is scope of further improvements

The scope of this push was to make it working to the point where project do

- successful installation 
- initiate a successful start with gulp
- and present the first screen as per the README file.
- and to store updates/changes remotely.